src/components/recs/team-tile.tsx

```diff
@@ -1,12 +1,13 @@
 import { teamIndexToPlayerData } from "@/server/teams";
-import { majorGodIndexToData } from "@/types/MajorGods";
+import { majorGodIndexToData, majorGodNameToData } from "@/types/MajorGods";
 import { IRecordedGame } from "@/types/RecordedGame";
-import { RecordedGamePlayerMetadata } from "@/types/RecordedGameParser";
+import { RecordedGamePlayerMetadata } from "@/types/recParser/RecordedGameParser";
 import Image from "next/image";
 
 function PlayerTile(playerData: RecordedGamePlayerMetadata, gameGuid: string) {
   const { name, civ, id } = playerData;
-  const godData = majorGodIndexToData(civ);
+  let godData = majorGodIndexToData(civ);
```

Comment:
Why was this changed to a `let`? Is this getting changed somewhere that I am not seeing?

Reply:
I missed this when reverting the frontend change here - I'll sort this

Reply:
let me know when you are able to update this